packages/client/src/generation/TSClient/Model.ts

```diff
@@ -54,20 +54,18 @@ export class Model implements Generatable {
   }
   protected get argsTypes(): Generatable[] {
     const { mapping, model } = this
```

Comment:
```suggestion
    const { mapping } = this
```
`model` is unused apparently